src/components/githubGoogleError_comp.js

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import closeIcon from "../../public/assets/githubGoogleError/closecircle.svg";
+
+import Image from "next/image";
+import errorIcons from "../../public/assets/githubGoogleError/error.svg";
+
+function GithubGoogleComponent() {
```

Comment:
This can have a better more descriptive name

Reply:
Done